draft-ietf-quic-qpack.md

```diff
@@ -1738,7 +1740,7 @@ Stream: Encoder
                               ^-- acknowledged --^
                                4   0  :authority  www.example.com
                                5   0  custom-key  custom-value2
-                              Size=215
+                              Size=214
```

Comment:
I attempted to count the difference between `:authoritywww.example.com` and `custom-keycustom-value2`.

Reply:
This looks right too, but it makes me want to re-run the examples through the code I wrote to verify (I made @martinthomson's changes by hand).

Reply:
Wait, let's go to the video tape!  I think 215 is correct.

I re-ran my decoder against the input, and it gets 215.

~~~
DecoderTable:

[1] (s=55) custom-key: custom-value2
[2] (s=57) :authority: www.example.com
[3] (s=54) custom-key: custom-value
[4] (s=49) :path: /sample/path
total size: 215
~~~
[Note: the decoder table prints with HPACK relative indices, hence they are 1 based and in reverse order]

Also verified in the shell:

~~~
# echo -n :authoritywww.example.com | wc -c
25
# echo -n custom-keycustom-value2 | wc -c
23
~~~

And third, I examined the table which has the two values in consecutive rows, and it appears only the "om" hangs over the 'custom-value2'.

Reply:
Well crap, I can only assume that I typo'd something when checking in the shell.  Sorry about that, and thanks for double-checking.